comm.go

```diff
@@ -428,6 +428,12 @@ func (com *Communication) estimateTpb(tpbChan chan<- int) {
 func (com *Communication) Communicate(txCurve map[int32]*Row, miner *Miner, actors []*Actor) {
 	defer com.wg.Done()
 
+	go func() {
```

Comment:
I think we can do without the goroutine since we're waiting for the blocks in the `for select` below anyway.